src/react/components/chartArrayLayout.css

```
@@ -0,0 +1,96 @@
+
+.mdv-chart-array[data-layout="grid"][data-grid-fill="true"] {
+    height: 100%;
+    min-height: 100%;
+    align-content: stretch;
+}
+
+.mdv-chart-array[data-layout="grid"] .mdv-chart-array__cell {
+    box-sizing: border-box;
+    flex: 1 1
+        calc(
+            (100% - var(--mdv-chart-array-gap) * (var(--mdv-chart-array-columns) - 1)) /
```

Stylelint (17.11.0)

[error] Unexpected newline after "/" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)